src/pybind/mgr/dashboard/openapi.yaml

```diff
@@ -3559,6 +3559,11 @@ paths:
                     items:
                       type: string
                     type: array
+                  service_instances:
+                    description: Service instances related to the host
+                    items:
+                      type: string
+                    type: array
```

Comment:
This is not correct. This should be [type `object`](https://swagger.io/docs/specification/data-models/data-types/#object) (and [JSON Schema `object`](https://json-schema.org/understanding-json-schema/reference/object.html)):
```yaml
type: object
additionalProperties:
  type: integer
```

Reply:
done this as well